src/http-client.js

```diff
@@ -123,6 +123,75 @@ export class HttpClient {
 
     return this::trackRequestEndWith(promise);
   }
+
+  /**
+  * Calls fetch as a GET request.
+  *
+  * @param input The resource that you wish to fetch. Either a
+  * Request object, or a string containing the URL of the resource.
+  * @param init An options object containing settings to be applied to
+  * the Request.
+  * @returns A Promise for the Response from the fetch request.
+  */
+  get(input: Request|string, init?: RequestInit): Promise<Response> {
+    return this.fetch(input, init);
+  }
+
+  /**
+  * Calls fetch with request method set to POST.
+  *
+  * @param input The resource that you wish to fetch. Either a
+  * Request object, or a string containing the URL of the resource.
+  * @param body The body of the request.
+  * @param init An options object containing settings to be applied to
+  * the Request.
+  * @returns A Promise for the Response from the fetch request.
+  */
+  post(input: Request|string, body?: any, init?: RequestInit): Promise<Response> {
+    return this::callFetch(input, body, init, 'post');
```

Comment:
As much as I love the bind operator, it's still a bit early for us to use this I think. Would you mind switching this to more standard JS?

Reply:
Actually, the bind operator is already being used a decent amount in this codebase. Here's an example: https://github.com/aurelia/fetch-client/blob/master/src/http-client.js#L107

Looks like it's used six times in the current http-client.js

Reply:
Doh. We should fix that then. I don't even see it on the proposals list anymore...